Docker/app.py

```diff
@@ -0,0 +1,58 @@
+from flask import Flask, send_from_directory, send_file, render_template, request, redirect, url_for
```

Comment:
app.py should not be copied in multiple places, like Docker, Ansible .... 

Consider the project structure as follows:

MyWeb/
  Docker/
     - Dockerfile
     - docker-compose.yaml
  Ansible/
     - only ansible related files
  MyWeb/
     - app.py
     - templates
     - requirements.txt